synapse/models/crypto.py

```diff
@@ -34,6 +34,20 @@ def getModelDefs(self):
                 ('crypto:smart:contract', ('guid', {}), {
                     'doc': 'A smart contract.',
                 }),
+                ('crypto:smart:effect:transfertoken', ('guid', {}), {
```

Comment:
The name is close to "transfertokens" and may get confusing. Recommend considering another secondary prop name like "transferowner".

Reply:
It's more about what you are transferring the ownership *of*.  This specific effect transfers 1 non-fungible token and the other transfers N fungible tokens. Open to name changes but "transferowner" is too general.